domains/tmasikt.json

```diff
@@ -0,0 +1,14 @@
+{
+  "description": "tmasikt's server record"
```

Comment:
missing a comma `,` after the description

Reply:
fixed